meta-resin-common/recipes-core/initrdscripts/files/machineid

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+# Copyright (C) 2011 O.S. Systems Software LTDA.
+# Licensed on MIT
+
+machineid_enabled() {
+  mount -t ext4 /dev/disk/by-label/resin-state $ROOTFS_DIR/mnt/state
```

Comment:
This script clearly depends on udev - so please add a rdepends on the package which includes this script.